melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/identifisering/BucIdentifiseringService.java

```diff
@@ -26,9 +28,10 @@
     @Transactional
     public void bucIdentifisert(String rinaSaksnummer, String aktoerId) {
         sedMottattHendelseRepository.findAllByPublisertKafka(false).stream()
-                .filter(it -> rinaSaksnummer.equals(it.getSedHendelse().getRinaSakId()))
+                .filter(hendelse -> rinaSaksnummer.equals(hendelse.getSedHendelse().getRinaSakId()))
+                .sorted(Comparator.comparing(SedMottattHendelse::getMottattDato))
```

Comment:
Pros og cons for å sortere her over i selve repository-kallet?

Reply:
👍  Var mening å gjøre dette. Bør også filtrere på `rinaSaksnummer` i db-kallet, som er litt mer tungvindt grunnet må slå opp i json-struktur, og trenger muligens å legge på en index også i så fall. Men ja, det bør gjøres 😄